presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParserErrorHandling.java

```diff
@@ -39,85 +39,85 @@
                 {"SELECT a FROM a AS x TABLESAMPLE x ",
-                        "line 1:34: mismatched input 'x'. Expecting: 'BERNOULLI', 'SYSTEM'"},
+                 "line 1:34: mismatched input 'x'. Expecting: 'BERNOULLI', 'SYSTEM'"},
                 {"SELECT a AS z FROM t GROUP BY CUBE (a), ",
-                        "line 1:41: mismatched input '<EOF>'. Expecting: '(', 'CUBE', 'GROUPING', 'ROLLUP', <expression>"},
+                 "line 1:41: mismatched input '<EOF>'. Expecting: '(', 'CUBE', 'GROUPING', 'ROLLUP', <expression>"},
                 {"SELECT a AS z FROM t WHERE x = 1 + ",
-                        "line 1:36: mismatched input '<EOF>'. Expecting: <expression>"},
+                 "line 1:36: mismatched input '<EOF>'. Expecting: <expression>"},
                 {"SELECT a AS z FROM t WHERE a. ",
-                        "line 1:31: mismatched input '<EOF>'. Expecting: <identifier>"},
+                 "line 1:31: mismatched input '<EOF>'. Expecting: <identifier>"},
                 {"CREATE TABLE t (x bigint) COMMENT ",
-                        "line 1:35: mismatched input '<EOF>'. Expecting: <string>"},
+                 "line 1:35: mismatched input '<EOF>'. Expecting: <string>"},
                 {"SELECT * FROM ( ",
-                        "line 1:17: mismatched input '<EOF>'. Expecting: '(', 'LATERAL', 'UNNEST', <identifier>, <query>"},
+                 "line 1:17: mismatched input '<EOF>'. Expecting: '(', 'LATERAL', 'UNNEST', <identifier>, <query>"},
                 {"SELECT CAST(a AS )",
-                        "line 1:18: mismatched input ')'. Expecting: <type>"},
+                 "line 1:18: mismatched input ')'. Expecting: <type>"},
                 {"SELECT CAST(a AS decimal()",
-                        "line 1:26: mismatched input ')'. Expecting: <integer>, <type>"},
+                 "line 1:26: mismatched input ')'. Expecting: <integer>, <type>"},
                 {"SELECT foo(*) filter (",
-                        "line 1:23: mismatched input '<EOF>'. Expecting: 'WHERE'"},
+                 "line 1:23: mismatched input '<EOF>'. Expecting: 'WHERE'"},
                 {"SELECT * FROM t t x",
-                        "line 1:19: mismatched input 'x'. Expecting: '(', ',', 'CROSS', 'EXCEPT', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LEFT', 'LIMIT', 'NATURAL', 'ORDER', " +
-                                "'RIGHT', 'TABLESAMPLE', 'UNION', 'WHERE', <EOF>"},
+                 "line 1:19: mismatched input 'x'. Expecting: '(', ',', 'CROSS', 'EXCEPT', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LEFT', 'LIMIT', 'NATURAL', 'ORDER', " +
+                         "'RIGHT', 'TABLESAMPLE', 'UNION', 'WHERE', <EOF>"},
                 {"SELECT * FROM t WHERE EXISTS (",
-                        "line 1:31: mismatched input '<EOF>'. Expecting: <query>"}};
+                 "line 1:31: mismatched input '<EOF>'. Expecting: <query>"}};
```

Comment:
end this line with `},` and put `};` on the next line -- you won't need to modify this line in order to add another test case later